pkg/runtime/runtime.go

```diff
@@ -93,6 +93,7 @@ type DaprRuntime struct {
 	namespace                string
 	scopedPublishings        []string
 	allowedTopics            []string
+	daprHTTPAPI              http.API
```

Comment:
nit: this can just be a bool, something like isHttpSerververStarted

Reply:
> nit: this can just be a bool, something like isHttpServerStarted

I need API object to call SetReadyStatus().